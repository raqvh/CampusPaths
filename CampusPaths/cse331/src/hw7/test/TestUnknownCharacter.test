#Test trying to find a path with a nonexistant character

LoadGraph csefriends CSEfriends.tsv

#First character is unknown
FindPath csefriends John Jack

#Second character is unknown
FindPath csefriends Raquel Sarah

#Both characters are unknown
FindPath csefriends John Sarah