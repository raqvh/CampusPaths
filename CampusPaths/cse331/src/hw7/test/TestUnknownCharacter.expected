#Test trying to find a path with a nonexistant character

loaded graph csefriends

#First character is unknown
unknown character John

#Second character is unknown
unknown character Sarah

#Both characters are unknown
unknown character John
unknown character Sarah